repair/repair_task.hh

```diff
@@ -250,8 +250,29 @@ protected:
 };
 
 class repair_module : public task_manager::module {
+    // TODO Deexie:
+    // _status - the difference between _status and modules maps that makes us keep this map is that the information about tasks that failed
+    // is kept forever (successful one are deleted after they are finished). Thus, we can keep the map as it is or keep a set of sequence numbers of failed
```

Comment:
can we erase the status after we returned the error to the user? and/or after the task ttl expires?

Reply:
This would change the existing repair api. Currently we can get the status of all tasks that happened in the past - the ones that failed are kept in `_status` forever.